src/editor/validator.ts

```diff
@@ -409,6 +410,20 @@ export class Validator {
         return false;
     }
 
+    /**
+     * logic: checks if rootNode is instanceof MethodCallModifier
+     */
+    isMethodCallModifierStatement(providedContext?: Context): boolean {
```

Comment:
what is this for?

Reply:
it is to check if the statement is a MethodCallModifier like this 
![image](https://user-images.githubusercontent.com/63805941/172916739-8a93155d-d935-44e4-b8bf-bf1246924ba4.png)
then we want to delete the entire statement when we click delete on the empty hole

Reply:
deleting inside the find hole should just delete the `.find()` not the entire statement